src/osa/scripts/tests/test_osa_scripts.py

```diff
@@ -198,6 +199,7 @@ def test_closer(
     for obs_file in test_observed_data:
         assert obs_file.exists()
     assert merged_run_summary.exists()
+    assert longterm_dir.exists()
```

Comment:
on top of longterm dir you also should create before the dir with all links. I suggest creating:

```python
@pytest.fixture(scope="session")
def longterm_link_latest_dir(base_test_dir):
    directory = base_test_dir / "OSA" / "DL1DataCheck_LongTerm" / to / expected / location
    directory.mkdir(parents=True, exist_ok=True)
    return directory
```
then also assert its existence in test_closer